### PR TITLE
Renovate Peer Review Documentation

### PR DESCRIPTION
**Pull request by mrdavidburns**
Added section to bottom of README.md describing how to peer review Renvoate PRs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -598,3 +599,51 @@ tugboat:php:init:
     - apt-get install -y libldap2-dev
     - docker-php-ext-install ldap
 ```
+
+Creating comprehensive documentation for conducting peer reviews on automated updates within the "Drainpipe" project is crucial to ensure consistency, maintainability, and the quality of the codebase. Here's a draft outline and content you could use or adapt for your documentation:
```
Comment (justafish) [inline README.md:603]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -598,3 +599,51 @@ tugboat:php:init:
...
+
+## Semantic Versioning and Update Types
+
+Drainpipe adheres to semantic versioning, categorizing updates into patches, minor, and major updates. Understanding these categories is crucial for the review process:
+
+- **Patch Releases (`x.y.Z`)**: Updates that fix bugs without adding new features or breaking existing functionality. If all tests pass, these updates are generally safe to merge.
+- **Minor Point Releases (`x.Y.z`)**: Updates that introduce new features without breaking backward compatibility. While these are often safe to merge if all tests pass, they require more scrutiny:
+  - Read the change logs carefully to understand the new features and fixes.
+  - Assess if the changes necessitate additional test coverage or could potentially impact existing functionality.
+  - Consider the implications of new features on the project's future development and maintenance.
```
Comment (justafish) [inline README.md:623]: I think this should go in a separate section as it's not really related to us internally accepting Renovate PRs

**Review by justafish — CHANGES_REQUESTED** [review]
Thanks for this @mrdavidburns !
I've left a few comments

File: README.md (hunk 2 above)
Comment (mrdavidburns) [reply README.md:622]: Maybe it's worded wrong? The point I was attempting to make here is that there are different levels of testing based on the type of update are being made. 

What if we just re-worded this to say 
```
All Renovate peer reviews regardless if they're a minor or patch release require:
- Reading the change logs carefully to understand the new features and fixes.
  - Assess if the changes necessitate additional test coverage or could potentially impact existing functionality.
  - Consider the implications of new features on the project's future development and maintenance.
- All tests and checks must pass
```

**Comment by deviantintegral** [comment]
Marking as high because this should enable more of the team to help review updates.

File: README.md (hunk 2 above)
Comment (mrdavidburns) [reply README.md:626]: I've updated this section with what I had suggested above. 

This is ready for review.

**Review by justafish — APPROVED** [review]

**Pull request merged**